react/week1/fibonanci/src/App.js

```diff
@@ -0,0 +1,24 @@
+import logo from "./logo.svg";
+import "./App.css";
+
+import React, { useState } from "react";
+import "./App.css";
+
+const App = () => {
+  const [state, setState] = useState([1, 1]);
```

Comment:
May be a separate component for Fibonacci inside the APP component would be nice.